1.17 templates cannot implements 1.15 interfaces (#929)
digital-asset/docs.daml.com: docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst

These restrictions are required to give a simple model of runtime
upgrades, avoiding ambiguity and non-obvious side effects. If you
require any of these types of changes, you may need to consider a
redeployment with downtime, either using the approach suggested in
:ref:`Rolling out backwards-incompatible changes
<backwards-incompatible-changes>` or using the existing upgrade

> procedure described in :ref:`here <upgrade-overview>`.

Reviewer: ```suggestion
procedure described in :ref:`Upgrade overview <upgrade-overview>`.
```

In this version of SCU, the following functionality has not yet been
implemented, but may be implemented in future releases.

> - A LF 1.17 templates cannot implement A LF 1.15 interfaces.

Reviewer 2: ```suggestion
- A LF 1.17 templates cannot implement a LF 1.15 interface.
```

- Retroactive interface instances are not compatible with SCU
  upgrades.  SCU allows instances to be changed in an
  upgrade. However, a new interface instance cannot be added to a

>   template in an upgrade; it requires an offline migration.

Reviewer: ```suggestion
  template in an upgrade. Adding a new interface instance requires an offline migration.
```
